rts/Lua/LuaVBOImpl.cpp

```diff
@@ -1486,6 +1486,17 @@ void LuaVBOImpl::DumpDefinition()
 	LOG("%s", ss.str().c_str());
 }
 
+/*** Gets the OpenGL Buffer ID
+ *
+ * @function VBO:GetID
+ * @treturn number buffer ID
```

Comment:
Should be:
```
@return number bufferID
```
Because `@return <type> <name> [Optional description.]`.

It's a bit odd that returns get names, but that's the standard. I think it helps in multi-return cases.